src/middleware/cache.js

```diff
@@ -1,10 +1,29 @@
-module.exports = (req, res, next) => {
-	res.FT_NO_CACHE = 'max-age=0, no-cache, must-revalidate';
-	res.FT_NO_CACHE_PRIVATE = 'max-age=0, no-cache, no-store, must-revalidate, private';
-	res.FT_SHORT_CACHE = 'max-age=600, stale-while-revalidate=60, stale-if-error=86400';
-	res.FT_HOUR_CACHE = 'max-age=3600, stale-while-revalidate=60, stale-if-error=86400';
-	res.FT_DAY_CACHE = 'max-age=86400, stale-while-revalidate=60, stale-if-error=86400';
-	res.FT_WEEK_CACHE = 'max-age=604800, stale-while-revalidate=60, stale-if-error=259200';
-	res.FT_LONG_CACHE = 'max-age=86400, stale-while-revalidate=60, stale-if-error=259200';
+/**
+ * @typedef {[key: keyof import("../../typings/n-express").CacheHeaders, val: string]} CacheHeader
+ */
+
+const cacheHeaders = {
+	FT_NO_CACHE: 'max-age=0, no-cache, must-revalidate',
+	FT_NO_CACHE_PRIVATE: 'max-age=0, no-cache, no-store, must-revalidate, private',
+	FT_SHORT_CACHE: 'max-age=600, stale-while-revalidate=60, stale-if-error=86400',
+	FT_HOUR_CACHE: 'max-age=3600, stale-while-revalidate=60, stale-if-error=86400',
+	FT_DAY_CACHE: 'max-age=86400, stale-while-revalidate=60, stale-if-error=86400',
+	FT_WEEK_CACHE: 'max-age=604800, stale-while-revalidate=60, stale-if-error=259200',
+	FT_LONG_CACHE: 'max-age=86400, stale-while-revalidate=60, stale-if-error=259200'
+};
```

Comment:
**thought**: i hate that these are just attached to `res` for no reason other than it's there. i'd love to eventually see these as exports instead, and replace these with getters that output a deprecation warning when used (probably using `util.deprecate`)